reference-implementation/lib/helpers.js

```diff
@@ -77,7 +77,7 @@ exports.PromiseInvokeOrNoop = (O, P, args) => {
   }
 };
 
-exports.PromiseInvokeOrFallbackOrNoop = (O, P1, args1, P2, args2) => {
+exports.PromiseInvokeOrFallback = (O, P1, args1, F, args2) => {
```

Comment:
Here F is an abstract op, right? Then we should say "PromiseInvokeOrPerformFallback" instead of "PromiseInvokeOrCallFallback"